taxonomies/packaging_shapes.txt

```diff
@@ -95,19 +116,21 @@ it:Lattina per bevande
 pt:Lata de bebida, latas de bebidas, lata de alumínio, latas de alumínio, lata de refrigerante, latas de refrigerante, lata de refrigerantes, latas de refrigerantes
 
 # Can for food, not canned drinks
+# Note: "canned", "conserve" : already used in the preservation taxonomy
```

Comment:
could you explain to me why we can't have same word in different taxonomies ? (I mean, in packaging_xxxx I understand, but there this is in preservation, isn't it ?

Reply:
@alexgarel It's because I create a combined "packaging" taxonomy that includes all packaging_* taxonomies + the preservation taxonomy.  That's because today in the packaging field, we have things like "Surgelés, pot en plastique, métal, bouteille".

Reply:
oh ! I see.